### ISSUE TITLE
Missing Labels in Icon Buttons: ADA Upgrade

### ISSUE THREAD

**Issue by kzadrozny** [uchicago-library/universalviewer#2]
# ADA Issues Summary
- [ ] Icon buttons in `<div class="headerPanel">` and `<div class="footerPanel min">` do not have labels
- [ ] Collapse/expand buttons in `<div class="leftPanel">` and `<div class="rightPanel">` do not have labels

## Icon buttons do not have labels
Documentation: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role
### Current Code
```
<div class="nextOptions">
   <button class="btn imageBtn next" title="Next Image" tabindex="0">
       <i class="uv-icon-next" aria-hidden="true"></i>
   </button>
   <button class="btn imageBtn last" title="Last Image" tabindex="0">
      <i class="uv-icon-last" aria-hidden="true"></i>
   </button>
</div>
```

### Suggested Code
```
<div class="nextOptions">
   <button class="btn imageBtn next" aria-label="Next Image">
       <i class="uv-icon-next" aria-hidden="true"></i>
   </button>
   <button class="btn imageBtn last" aria-label="Last Image">
       <i class="uv-icon-last" aria-hidden="true"></i>
   </button>
</div>
```

## Collapse/expand buttons in sidebars
Would be nice if the buttons to toggle the sidebars had labels indicating that this is what the buttons do.
### Current Code
```
<div class="closed" aria-hidden="false" style="display: block;">
   <a class="expandButton" tabindex="0" title="Expand Information"></a>
   <a class="title" title="More Information">More Information</a>
</div>
```

### Suggested Code
```
<div class="closed" aria-hidden="false" style="display: block;">
   <a class="expandButton" tabindex="0" aria-label="Expand Information Sidebar"></a>
   <a class="title" title="More Information">More Information</a>
</div>
```

## Next Steps
- @verbalhanglider will submit an issue to the UV repo as a bug fix.
- Accessibility upgrades will be worked on in Phase 3 of the project.

**Comment by kzadrozny** [comment]
An extra item that I think we can solve in-house is: No "skip navigation" link.
Add a skip navigation link that allows user to go directly to the main content (the object image pane).
Documentation: https://webaim.org/techniques/skipnav/
